### ISSUE TITLE
Coding interviews

### ISSUE THREAD

**Issue by ibnemuhammad** [alexbunardzic/AMA#1]
How are coding interviews going?

**Comment by alexbunardzic** [comment]
Still trying to find the time to organize the next rounds. There is always something more pressing that comes up...

**Comment by alirafique99** [comment]
Ok, I am interested in having a go at it when you start with the next rounds.  Was it the type of question you were expecting here?
Or is this for more engineering-focused questions?

**Comment by alexbunardzic** [comment]
No, that's the perfect type of questions I'm hoping to get. It's ask me ANYTHING.
